app/assets/javascripts/cascade/level/news-events.js

```diff
@@ -1,407 +1,450 @@
 $(function () {
```

Comment:
Fix is in this file. I also cleaned up formatting so diff here is useless. Find relevant changes here:

- https://github.com/chapmanu/cascade-assets/pull/161/commits/072df4de9071a6069ab9592f4a529f7a42708930